src/utils/types/equipmentType.ts

```diff
@@ -78,13 +78,18 @@ export enum EquipmentType {
     BREAKER = 'BREAKER',
 }
 
-export enum HvdcType {
-    LCC = 'LCC',
-    VSC = 'VSC',
+export enum ExtendedEquipmentType {
```

Comment:
Why ExtendedEquipmentType ? In EquipmentType, we have HVDC_CONVERTER_STATION, VSC_CONVERTER_STATION and LCC_CONVERTER_STATION. Is it a problem to put HVDC_LINE_LCC and HVDC_LINE_VSC in EquipmentType ?